integration/src/test/java/org/broadleafcommerce/core/payment/service/NullIntegrationGatewayTransactionServiceImpl.java

```diff
@@ -158,9 +160,9 @@ protected PaymentResponseDTO commonCreditCardProcessing(PaymentRequestDTO reques
                 String expMonth = parsedDate[0];
                 String expYear = parsedDate[1];
                 try {
-                    DateTime expirationDate = new DateTime(Integer.parseInt("20" + expYear), Integer.parseInt(expMonth), 1, 0, 0);
-                    expirationDate = expirationDate.dayOfMonth().withMaximumValue();
-                    validDate = expirationDate.isAfterNow();
+                    Date expirationDate = new Date(Integer.parseInt("20" + expYear), Integer.parseInt(expMonth), 1, 0, 0);
```

Comment:
Looks like this is deprecated. Could you update this please?
![image](https://user-images.githubusercontent.com/35038090/195394187-455de61b-5957-436f-99d2-bd824228618c.png)